econet_heatpump_water_heater.yaml

```diff
@@ -10,6 +10,9 @@ substitutions:
   external_components_source: github://esphome-econet/esphome-econet@main
   logger_level: WARN
   econet_update_interval: 30s
+  # The following has been found to give more accurate results but is likely device specific
+  # sensor_power_filters_lambda: return x * 0.91 - 20;
+  sensor_power_filters_lambda: return x;
```

Comment:
Where is this sensor_power_filters_lambda defined? Can you add a comment linking to the details of why this improves things? Right now, it just looks like magic.

Reply:
line 113